spec/acceptance/deprecation_spec.rb

```diff
@@ -1,19 +1,28 @@
 #! /usr/bin/env ruby -S rspec
 require 'spec_helper_acceptance'
-require 'shellwords'
 
 describe 'deprecation function' do
-  before :each do
-    FileUtils.rm_rf '/tmp/deprecation'
+
+  if fact('operatingsystem') == 'windows'
+    test_file = 'C:/deprecation'
+    else
+    test_file = "/tmp/deprecation"
+  end
+
+  add_file_manifest = "\"deprecation('key', 'message') file { '#{test_file}': ensure => present, content => 'test', }\""
```

Comment:
It'd be totally easier to read and maintain that way, but since it's run on the command line (puppet apply -e style) instead of with apply_manifest, Windows seems to need it formatted this way.